src/types/Config.tsx

```diff
@@ -110,28 +120,24 @@ export const ALL_PAGES: IPage[] = [
     name: PAGES.DOCUMENTATION,
     element: <Redirect path="documentation" tab={'documentation'} />,
   },
-  {
-    name: PAGES.STORYBOOK,
-    element: <Redirect path="_storybook" tab={'storybook'} />,
-  },
   {
     name: PAGES.MARKETPLACE,
-    role: ROLES.APPSTORE_VIEW,
+    allowTo: () => userHasPortalRole(ROLES.APPSTORE_VIEW),
```

Comment:
@oyo could you please recheck what APPSTORE_VIEW is? I dont think we have such an role. The actual permission name is `view_apps`